### ISSUE TITLE
Move this repo to mozilla-services org

### ISSUE THREAD

**Issue by pdehaan** [pdehaan/speaktome-loadtests#11]

**Comment by rpappalax** [comment]
if i'm not mistaken the process is: (1) to transfer ownership to mozilla-services in settings (2) ask @karlht to give us ownership permission of the repo once it belongs to mozilla-services

**Comment by pdehaan** [comment]
Yeah, that was one of the reasons i was dragging my feet. I always recall once I transfer, i lose the ability to change descriptions or URLs unless i get elevated permissions.

Also, I'll probably put these in **mozilla/** instead of **mozilla-services/** so they live adjacent to the actual code repos. I only put **screenshots-loadtests** in **mozilla-services** since that's where the code lived.

**Comment by karlht** [comment]
I concur with Peter here; it feels more important to me that the tests live near the code than that they live in mozilla-services.

**Comment by rpappalax** [comment]
Agreed!
